product_docs/docs/biganimal/release/pricing_and_billing/index.mdx

```diff
@@ -26,6 +26,8 @@ If you're connecting directly to BigAnimal through BigAnimal's cloud account, ED
 
 If you're connecting directly to BigAnimal through your Microsoft Azure or AWS account, EDB doesn't bill you for cloud infrastructure, such as compute, storage, data transfer, monitoring, and logging.
```

Comment:
This line needs to be aligned with line 31. We should discuss charges for BYOA in a combined section

Reply:
I split the Cloud infrastructure section into two: BigAnimal's cloud account and Microsoft Azure or AWS account. Does that work?

Reply:
It's now BigAnimal's cloud account and Using your own cloud account.